res/values/arrays.xml

```diff
@@ -153,6 +153,18 @@
       <item>none</item>
   </string-array>
 
+  <string-array name="pref_show_emails_entries">
+    <item>@string/off</item>
+    <item>@string/pref_show_emails_accepted_contacts</item>
+    <item>@string/pref_show_emails_all</item>
+  </string-array>
+
+  <string-array name="pref_show_emails_values">
+    <item>0</item>
+    <item>1</item>
+    <item>3</item>
```

Comment:
why not 2? ;)

Reply:
i'm using bits to be prepared when we might want to separate "show mua-replies in chats" (bit#0) from "show all emails in an activity" (bit#1). in the core headers, these are named values, it's a bit ugly here, not sure if the xml stuff can use values from the java/native part.

Reply:
as "show emails in an activity" is now always on again, there seems to be very few reasons left not to use `2` here. doing a commit.